internal/upload/koji/koji_test.go

```diff
@@ -22,17 +24,36 @@ import (
 
 func TestKojiImport(t *testing.T) {
 	// define constants
-	server := "http://localhost:8080/kojihub"
-	user := "osbuild"
-	password := "osbuildpass"
+	server := "https://localhost/kojihub"
 	filename := "image.qcow2"
 	filesize := 1024
+	shareDir := "/tmp/osbuild-composer-koji-test"
```

Comment:
Use a `TmpDir()` to avoid clashes with other test runs?

Reply:
I think I don't care for now, this tempdir is shared between the test and the container runner. Sharing the tempdir path would be more pain than I currently want.